### PR TITLE
fix: Projects must contain a supported web or test framework

### PR DESCRIPTION
**Pull request by dustinbyrne**
Solely relying on language support is not possible at the moment. A web or test framework is required for triggering recordings on and off.

### CODE REVIEW

**Review by kgilpin — COMMENTED** [review]
Can we use a softer message like - installer doesn’t recognize a framework or test thingy. Follow manual install instructions for best results.

File: packages/components/src/components/install-guide/ProjectPickerRow.vue [hunk 1]
```diff
@@ -46,13 +46,13 @@
...
           <p>Finished the installation? Proceed to the next step.</p>
-          <v-navigation-buttons :first="true" />
+          <v-navigation-buttons :first="true" :last="!supported" />
         </div>
       </template>
       <template v-else>
         <p>
-          This project is not supported by AppMap. For a list of supported integrations, visit our
-          documentation:
+          This project is not supported by AppMap. For a list of all supported integrations, visit
```
[the remark this answers is not in the document]
Comment (kgilpin) [reply packages/components/src/components/install-guide/ProjectPickerRow.vue:54]: The project picker is useful but we have to be really careful about rejecting users.

[the remark this answers is not in the document]
Comment (ahtrotta) [reply packages/components/src/components/install-guide/ProjectPickerRow.vue:54]: Maybe softer language: `It doesn't look like this project is supported by AppMap. For a list of all supported integrations, visit our documentation:`

[the remark this answers is not in the document]
Comment (dustinbyrne) [reply packages/components/src/components/install-guide/ProjectPickerRow.vue]: Thanks for the feedback. I cleaned up the language a bit - we'll inform the user we didn't find a web or test framework and point them to the appropriate documentation for their language.

image

**Comment by dustinbyrne** [comment]
These changes have been rolled into https://github.com/getappmap/appmap-js/pull/748

**Pull request closed without merging**